cmd/clusterctl/cmd/init.go

```diff
@@ -100,6 +103,10 @@ func init() {
 		"Control plane providers and versions (e.g. kubeadm:v0.3.0) to add to the management cluster. If unspecified, the Kubeadm control plane provider's latest release is used.")
 	initCmd.Flags().StringVar(&initOpts.targetNamespace, "target-namespace", "",
 		"The target namespace where the providers should be deployed. If unspecified, the provider components' default namespace is used.")
+	initCmd.Flags().BoolVar(&initOpts.waitProviders, "wait-providers", false,
+		"Wait for providers to be installed.")
+	initCmd.Flags().IntVar(&initOpts.waitProvidersTimeout, "wait-providers-timeout", 30,
```

Comment:
```suggestion
	initCmd.Flags().DurationVar(&initOpts.waitProvidersTimeout, "wait-providers-timeout", "30s",
```

Reply:
Let set this to 5minutes, so we account for slow machines/network